spotify-recent-plays/test/Connector.test.js

```diff
@@ -0,0 +1,407 @@
+      {
+        "values": [
+          "Get Better",
+          "Nothing But Thieves"
+        ]
+      },
+      {
+        "values": [
+          "Careless",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "War",
+          "grandson"
+        ]
+      },
+      {
+        "values": [
+          "What Makes A Good Man?",
+          "The Heavy"
+        ]
+      },
+      {
+        "values": [
+          "I Sat By The Ocean",
+          "Queens of the Stone Age"
+        ]
+      },
+      {
+        "values": [
+          "Wolf",
+          "Highly Suspect"
+        ]
+      },
+      {
+        "values": [
+          "She's Creeping",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Little Monster",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Come On Over",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Out Of The Black - Live From Spotify NYC",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Loose Change - Live From Spotify NYC",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Figure It Out - Live From Spotify NYC",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Come On Over - Live From Spotify NYC",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Figure It Out",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Lights Out",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Hook, Line & Sinker",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Birthday Sex",
+          "Jeremih"
+        ]
+      },
+      {
+        "values": [
+          "I Only Lie When I Love You",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "My Sharona - Recorded at Abbey Road Studios, London",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "How Did We Get So Dark? - Recorded at Abbey Road Studios, London",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Better Strangers",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Ten Tonne Skeleton",
+          "Royal Blood"
+        ]
+      },
+      {
+        "values": [
+          "Careless",
+          "Royal Blood"
+        ]
+      }
+    ],
+    "schema": [
+      {
+        "dataType": "STRING",
+        "label": "Track Name",
+        "name": "track_name",
+        "semantics": {
+          "conceptType": "DIMENSION"
+        }
+      },
+      {
+        "dataType": "STRING",
+        "label": "Artist",
+        "name": "artist",
+        "semantics": {
+          "conceptType": "DIMENSION"
+        }
+      }
+    ]
+  };
```

Comment:
Could you par down this test data a bit, it's size sorta gets in the way of what it's testing.